bin/spack

```diff
@@ -43,6 +43,11 @@ sys.path.insert(0, spack_lib_path)
 
 # Add external libs
 spack_external_libs = os.path.join(spack_lib_path, "external")
+
+if sys.version_info[:2] == (2, 6):
+    sys.path.insert(0, os.path.join(spack_external_libs, 'py26'))
+    import ordereddict
```

Comment:
Do we have to import this in this file? It will likely slow things down a bit.

Reply:
FWIW the statement `import ordereddict` can be removed after the patch at https://github.com/spack/spack/pull/6931#discussion_r161347577. I think it has something to do with the naming conflicts introduced by adding the module `spack.util.ordereddict`.